formbricks/formbricks
# docs: how we code documentation

## What does this PR do?



Adds **How we code at Formbricks** to our documentation 

https://github.com/formbricks/formbricks/assets/31730715/d89fbe29-3f67-4c98-bfde-6a8ef15ea6ad


## Type of change




## How should this be tested?



- Lauch application
- Go to docs
- Go to contribution section
- Click **How we code at Formbricks**
- You should see a list of rules where each section have unique links

## Checklist
